subsys/nfc/tnep/Kconfig

```diff
@@ -27,6 +28,13 @@ config NFC_TNEP_RX_MAX_RECORD_SIZE
 	help
 	  Set the maximum size of received NDEF Record
 
+config NFC_TNEP_ENCODE_NLEN
```

Comment:
Do you really need this configuration? You could simply check if T4T stack configuration is enabled

Reply:
It is not necessary to have this additional configuration. I changed this to use the T4T config from the nrfxlib